tests/openvino/native/models.py

```diff
@@ -210,6 +210,19 @@ def _create_ov_model(self):
         return model
 
 
+class FP16InputModel(OVReferenceModel):
```

Comment:
Why not use/modify the existing `FPModel` above?

Reply:
Added `ìnput_dtype` argument to `FPModel`